NEWS.md

```diff
@@ -102,6 +165,25 @@ contributors as noted in the log.
 
 ### Fixed
 
+- Notify idle manager when emulating cursor move (forgotten in original
+  implementation).
+- Fix GrowToEdge/ShrinkToEdge action bug caused by clients ignoring the
+  requested size, for example a terminal honouring size-hints.
+- Fix `assert()` on VT switch. Issue #1667
+- Use pre-multiplied colors by default to ensure consistent colors for
+  titlebars with transparency (because the corner pieces uses different
+  technology to the remainder of the titlebar). #1684
```

Comment:
This feels like too much implementation details, suggest something like "Ensure the titlebar has a consistent look when using transparency".